spring-cloud-gcp-bigquery/src/test/java/com/google/cloud/spring/bigquery/integration/outbound/BigQueryFileMessageHandlerIntegrationTests.java

```diff
@@ -59,19 +60,25 @@
 @DirtiesContext(classMode = ClassMode.AFTER_EACH_TEST_METHOD)
 class BigQueryFileMessageHandlerIntegrationTests {
 
-  private static final String TABLE_NAME = "test_table";
+  private String tableName;
 
-  @Autowired private ThreadPoolTaskScheduler taskScheduler;
+  @Autowired
+  private ThreadPoolTaskScheduler taskScheduler;
 
-  @Autowired private BigQuery bigquery;
+  @Autowired
+  private BigQuery bigquery;
 
-  @Autowired private BigQueryFileMessageHandler messageHandler;
+  @Autowired
+  private BigQueryFileMessageHandler messageHandler;
 
   @BeforeEach
   @AfterEach
   void setup() {
+    if (tableName == null) {
+      tableName = "test_table_" + UUID.randomUUID().toString().replace('-', '_');
+    }
     // Clear the previous dataset before beginning the test.
-    this.bigquery.delete(TableId.of(DATASET_NAME, TABLE_NAME));
+    this.bigquery.delete(TableId.of(DATASET_NAME, tableName));
```

Comment:
Why delete the table before running the test? It shouldn't exist.

Reply:
This method is running before and after each test and the delete is meaningful only after the test.